projects/plugins/wpcomsh/plugin-hotfixes.php

```diff
@@ -138,3 +138,30 @@ function get_magic_quotes_gpc() {
 		return false;
 	}
 }
+
+/**
+ * Use filter to remove the Elementor Developer Notice.
+ *
+ * Gutenberg is a default plugin installed on all WordPress.com sites, 
+ * however, its presence triggers the beta install notice to appear even
+ * on production sites. 
+ *
+ * @param array $notices The array of admin notices from Elementor.
+ * @return array Modified array without the Elementor Developer Notice.
+ */
+function wpcomsh_remove_elementor_dev_notice( $notices ) {
+
+    if ( is_array( $notices ) ) {
+        if ( class_exists( 'Elementor\Core\Admin\Notices\Elementor_Dev_Notice' ) ) {
```

Comment:
Maybe it's better to disable the autoload by passing the `false` to the second argument